### PR TITLE
Add more description and style to landing

### PR DESCRIPTION
**Pull request by mpetrunic**

### CODE REVIEW

File: src/pages/index.html [hunk 1]
```diff
@@ -101,11 +101,16 @@ <h3 class="title is-3 is-spaced is-title-reveal">How it works</h3>
...
 
-<section id="table" class="section section-light-grey is-medium">
+<section id="load-balancers" class="section section-light-grey is-medium">
     <div class="container">
             <div class="title-wrapper has-text-centered">
                 <h2 class="title is-2">Load balancers</h2>
                 <div class="divider is-centered"></div>
+
+                <p class="pt-20">
+                   List of publicly available Vedran load balancers. If you are hosting Vedran load balancer,
```
Comment (morrigan) [inline src/pages/index.html]: except the typo for instructions I would rephrase this: `If you are hosting Vedran load balancer and want to be on this list, see the editing instructions.`

**Review by morrigan — COMMENTED** [review]
Just this here from the changes but found few typos that I will fix as that's easier. 
EDIT: Nevermind, I thought some of those changes are on master already

File: src/partials/setupInstructions.html [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+<section id="setupInstructions" class="section setup is-medium  section-primary" data-color="#4FC1EA"
+    data-color-opacity=".6">
     <div class="container">
         <div class="title-wrapper has-text-centered">
-            <h2 class="title is-2 light-text is-spaced pb-40">Setup instructions</h2>
+            <h2 class="title is-2 light-text is-spaced">Setup instructions</h2>
+            <div class="divider is-centered"></div>
+
+            <p class="light-text pb-20 pt-20">
+                Help us decentralize infrastructure by either hosting public load balanacer
```
Comment (morrigan) [inline src/partials/setupInstructions.html]: balanacer => balancer

Comment (morrigan) [reply src/partials/setupInstructions.html]: the existing one as a node operator

File: src/partials/setupInstructions.html [hunk 3]
```diff
@@ -16,100 +23,196 @@ <h2 class="title is-2 light-text is-spaced pb-40">Setup instructions</h2>
...
-                        <span class="step">2.</span>
-                        <div class="code">
-                                ./vedran-daemon --id=<span class="hightlight">&#60;some secret id for your node&#62;</span> --lb=<span class="hightlight">&#60;load balancer url&#62;</span>
-                                --payout-address=<span class="hightlight">&#60;polkadot account address&#62;</span>
+                <div class="columns  pb-20">
+                    <div class="tile column is-half is-vertical">
+                        <h3 class="subtitle is-4">Vedran Load Balancer</h3>
+                        <div class="pb-20" style="margin-left: 0.5rem;">
+                            <span class="box is-size-6" style="width: 70%;">
+                                Use this for hosting public endpoint and distributing incomming requests
```
Comment (morrigan) [inline src/partials/setupInstructions.html]: incomming => incoming

File: src/partials/setupInstructions.html [hunk 4]
```diff
@@ -16,100 +23,196 @@ <h2 class="title is-2 light-text is-spaced pb-40">Setup instructions</h2>
...
-                <ul>
-                    <li><span class="step">1.</span> Download latest release: <a href="https://github.com/NodeFactoryIo/vedran-daemon/releases">Github</a></li>
-                    <li class="codeContainer">
-                        <span class="step">2.</span>
-                        <div class="code">
-                                ./vedran-daemon --id=<span class="hightlight">&#60;some secret id for your node&#62;</span> --lb=<span class="hightlight">&#60;load balancer url&#62;</span>
-                                --payout-address=<span class="hightlight">&#60;polkadot account address&#62;</span>
+                <div class="columns  pb-20">
+                    <div class="tile column is-half is-vertical">
+                        <h3 class="subtitle is-4">Vedran Load Balancer</h3>
```
Comment (morrigan) [inline src/partials/setupInstructions.html:28]: Maybe makes more sense to introduce LB in one sentence first and then continue with "Use this..." ?

File: src/partials/setupInstructions.html [hunk 5]
```diff
@@ -16,100 +23,196 @@ <h2 class="title is-2 light-text is-spaced pb-40">Setup instructions</h2>
...
+                            <li><span class="step">4.</span> Use <span class="code">--help</span> for additional
+                                configuration and flags</li>
+                        </ul>
+                    </div>
+
+                    <div class="tile column is-half is-vertical">
+                        <h3 class="subtitle is-4">Vedran Daemon</h3>
+                        <div class="pb-20" style="margin-left: 0.5rem;">
+                            <span class="box is-size-6" style="width: 70%;">
+                                Ran this alongside your Substrate based node to join
```
Comment (morrigan) [inline src/partials/setupInstructions.html]: Ran => run

File: src/partials/setupInstructions.html [hunk 6]
```diff
@@ -16,100 +23,196 @@ <h2 class="title is-2 light-text is-spaced pb-40">Setup instructions</h2>
...
-                    <li class="codeContainer">
-                        <span class="step">2.</span>
-                        <div class="code">
-                                ./vedran-daemon --id=<span class="hightlight">&#60;some secret id for your node&#62;</span> --lb=<span class="hightlight">&#60;load balancer url&#62;</span>
-                                --payout-address=<span class="hightlight">&#60;polkadot account address&#62;</span>
+                <div class="columns  pb-20">
+                    <div class="tile column is-half is-vertical">
+                        <h3 class="subtitle is-4">Vedran Load Balancer</h3>
+                        <div class="pb-20" style="margin-left: 0.5rem;">
+                            <span class="box is-size-6" style="width: 70%;">
```
Comment (morrigan) [inline src/partials/setupInstructions.html:91]: Can we reuse this description as a component? same typos here :D

File: src/partials/setupInstructions.html [hunk 7]
```diff
@@ -16,100 +23,196 @@ <h2 class="title is-2 light-text is-spaced pb-40">Setup instructions</h2>
...
+                            <li><span class="step">4.</span> Use <span class="code">--help</span> for additional
+                                configuration and flags</li>
+                        </ul>
+                    </div>
+
+                    <div class="tile column is-half is-vertical">
+                        <h3 class="subtitle is-4">Vedran Daemon</h3>
+                        <div class="pb-20" style="margin-left: 0.5rem;">
+                            <span class="box is-size-6" style="width: 70%;">
+                                Ran this alongside your Substrate based node to join
```
Comment (morrigan) [inline src/partials/setupInstructions.html]: ran => run

File: src/partials/setupInstructions.html [hunk 8]
```diff
@@ -16,100 +23,196 @@ <h2 class="title is-2 light-text is-spaced pb-40">Setup instructions</h2>
...
+                                    --private-key=<span class="hightlight">&#60;private seed from step 1.&#62;</span>
+                                </div>
+                            </li>
+                            <li><span class="step">4.</span> Use <span class="code">--help</span> for additional
+                                configuration and flags</li>
+                        </ul>
+                    </div>
+
+                    <div class="tile column is-half is-vertical">
+                        <h3 class="subtitle is-4">Vedran Daemon</h3>
```
Comment (morrigan) [inline src/partials/setupInstructions.html:123]: Would also love to see here 1 sentence introduction of what daemon is.

File: src/partials/setupInstructions.html (hunk 6 above)
Comment (mpetrunic) [reply src/partials/setupInstructions.html:91]: we could but it seems like too much effort 😅
